x-pack/plugin/core/src/main/resources/security-index-template-7.json

```diff
@@ -213,8 +213,17 @@
               "type": "date",
               "format": "epoch_millis"
             },
-            "superseded_by": {
-              "type": "keyword"
+            "superseding_access_token": {
```

Comment:
I'm up for suggestion. I'm not particularly fond of this. It doesn't need to be binary by the way, we can make it text but not index it I guess,

Reply:
I don't see any problem with keeping binary data in a binary type, and I agree that we should set `indexed: false`.

Reply:
> I don't see any problem with keeping binary data in a binary type, and I agree that we should set indexed: false.

I believe binary fields are not indexed/stored [by default](https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html) : 

> I don't mind the binary, but I would prefer a nested object rather than having 4 fields 

Keeping binary and will replace this with 
```
"superseding": {
              "type": "object",
              "properties": {
                "access_token": {
                  "type": "binary"
                },
                "refresh_token": {
                  "type": "binary"
                },
                "encryption_iv": {
                  "type": "binary"
                },
                "encryption_salt": {
                  "type": "binary"
                }
              }
            },
```